### PR TITLE
main/zfs: upgrade to 0.7.13

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

**Comment by HRio** [comment]
@TBK please rebase.

**Comment by TBK** [comment]
@HRio Done.

File: main/zfs-vanilla/APKBUILD [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 # when changing _ver we *must* bump _rel
 _name=zfs
-_ver=0.7.12
-_rel=0
+_ver=0.7.13
+_rel=1
```
Comment (HRio) [inline main/zfs-vanilla/APKBUILD]: _rel=0?

File: main/zfs/APKBUILD [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 url="http://zfsonlinux.org"
 arch="all !armhf !armv7"
 license="CDDL"
 depends_dev="glib-dev e2fsprogs-dev util-linux-dev libtirpc-dev attr-dev"
 makedepends="$depends_dev automake autoconf libtool linux-headers"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-dracut::noarch
+subpackages="$pkgname-dev $pkgname-doc $pkgname-openrc $pkgname-libs $pkgname-dracut::noarch
 	$pkgname-udev $pkgname-scripts $pkgname-utils-py:utils_py:noarch"
 source="https://github.com/zfsonlinux/zfs/releases/download/zfs-$pkgver/zfs-$pkgver.tar.gz
 	zfs-import-should-be-before-swap.patch
```
Comment (HRio) [inline main/zfs/APKBUILD]: zfs-import-should-be-before-swap.patch is missing from sha512sums

Comment (TBK) [reply main/zfs/APKBUILD]: I have corrected the mistake.

**Comment by HRio** [comment]
@ncopa can you help in reviewing this? Thanks

**Pull request closed without merging**